docker/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-ARG BASE=zilliqa/scilla:v0.10.0-alpha.1
+ARG BASE=zilliqa/scilla:7652ad30
```

Comment:
TODO : Rollback the image version which points from the current master scilla branch to the old one.